app/models/my_modules_repository_row.rb

```diff
@@ -0,0 +1,8 @@
+class MyModulesRepositoryRow < ActiveRecord::Base
```

Comment:
I still think you should change this to `MyModuleRepositoryRow`, just to be consistent with the rest of the codebase, no other reason.